spec/vcr_setup.rb

```diff
@@ -11,6 +11,19 @@
   c.filter_sensitive_data('D360-API-KEY') do |interaction|
     interaction.request.headers['D360-Api-Key'].try(:first)
   end
+  c.filter_sensitive_data('THREE_SIXTY_DIALOG_PARTNER_PASSWORD') do
+    ENV['THREE_SIXTY_DIALOG_PARTNER_PASSWORD']
+  end
+  c.filter_sensitive_data('THREE_SIXTY_DIALOG_PARTNER_USERNAME') do
+    ENV['THREE_SIXTY_DIALOG_PARTNER_USERNAME']
+  end
+  c.filter_sensitive_data('THREE_SIXTY_DIALOG_PARTNER_ID') do
+    ENV['THREE_SIXTY_DIALOG_PARTNER_ID']
+  end
+  c.filter_sensitive_data('Bearer <TOKEN>') do |interaction|
+    interaction.request.headers['Authorization'].try(:first)
```

Comment:
Nice. This works?

Reply:
Yes, I found this here https://stackoverflow.com/questions/71163339/replace-sensitive-data-in-vcr-gem-cassettes-request-body and other places. If you look at the cassette `vcr_cassettes/three_sixty_dialog_welcome_message_created.yml`, you can see that it filters out the token from the subsequent requests to fetch existing templates and create the new one.